use/extension.go

```diff
@@ -56,27 +56,32 @@ func (s *Statement) Emit(out io.Writer) {
 // Origin parses origin from reference
```

Comment:
Being a bit picky, but the changes in this file are unrelated to the PR, aren't they?